src/blosc2/__init__.py

```diff
@@ -328,6 +328,7 @@ class Tuner(Enum):
     arctan,
     arctan2,
     arctanh,
+    array_from_ffi_ptr,
```

Comment:
Better move this to the https://github.com/Blosc/python-blosc2/blob/main/src/blosc2/__init__.py#L106 block.

Reply:
No, I think the current is the proper place, so no worries.